settings.gradle.kts

```diff
@@ -1,3 +1,22 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        mavenCentral()
+    }
+}
+
+plugins {
+    id("com.gradle.enterprise") version "3.12.3"
+}
+
+gradleEnterprise {
+    buildScan {
+        termsOfServiceUrl = "https://gradle.com/terms-of-service"
+        termsOfServiceAgree = "yes"
+        publishAlways()
+    }
+}
+
```

Comment:
I am not sure if I want this in this repo - I would like to keep this minimal and you don't necessarily need this (although I am very much in favor for doing something like this in a build).

There was also some "unhappiness" about having a public repo automatically publish scans before:
https://github.com/jjohannes/gradle-project-setup-howto/issues/2

Can you either remove this commit from the PR and we either:
- Don't add this at all for now
- Or create a separate PR so we can discuss (and/or refine) it

Reply:
Dropped.